src/d3a/models/area/__init__.py

```diff
@@ -252,8 +252,6 @@ def _cycle_markets(self, _trigger_event=True, _market_cycle=False, deactivate=Fa
 
         # area_market_stats have to updated when cycling market of each area:
         self.stats.update_area_market_stats()
-        self.stats.aggregate_exported_imported_energy(self)
-
```

Comment:
I am not sure whether this is even worth mentioning. But i would prefer this to stay here and not to trigger it in the endpointbuffer as these are area related results. Maybe Spyros has a different opinion.

Reply:
The reason I moved it to `SimulationEndpointBuffer` class is that markets are not cycled at the end of simulation, so the last market after the end of simulation was missing with imported/exported energy stats.

Reply:
I would actually be of a third opinion, why is there a need to be called from either place ? Could this not be part of the `update_area_market_stats` method ? In general it will work as it is, but it is kind of strange to call the area stats update from the endpoint buffer, and it is also strange to call 2 methods from the AreaStats in the same place, which, judging by their name, look similar.